cmake/External/rccl.cmake

```diff
@@ -3,7 +3,7 @@ if(NOT __NCCL_INCLUDED)
 
   if(USE_SYSTEM_NCCL)
     # NCCL_ROOT, NCCL_LIB_DIR, NCCL_INCLUDE_DIR will be accounted in the following line.
-    find_package(RCCL REQUIRED)
+    find_package(rccl REQUIRED)
     if(RCCL_FOUND)
```

Comment:
```suggestion
    if(rccl_FOUND)
```

Reply:
Perhaps try this change.  I saw in the CI logs `RCCL NOT Found!` so that might be why we ran into linker errors later.

Reply:
ah good catch!